packages/cells/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": "@srcbook/configs/ts/base.json",
+  "compilerOptions": {
+    "noEmit": false,
+    "allowImportingTsExtensions": false,
+    "outDir": "dist",
+    "jsx": "react-jsx",
+    "baseUrl": ".",
+    "paths": {
+      "@/*": ["src/*"]
+    }
+  },
+  "include": ["index.tsx", "src/**/*.{ts.tsx}"],
+  "exclude": ["node_modules", "dist"]
+}
```

Comment:
The typescript configuration ended up being a hybrid between the `@srcbook/shared` file and the `@srcbook/web` file. Given this is the first web-specific package, let me know if there's any tweaks that should be made here.